src/tv2-common/actions/executeAction.ts

```diff
@@ -1837,6 +1847,59 @@ function executeActionRecallLastDVE<
 	}
 }
 
+function executeActionPlayGraphics<
+	StudioConfig extends TV2StudioConfigBase,
+	ShowStyleConfig extends TV2BlueprintConfigBase<StudioConfig>
+>(
+	context: ITV2ActionExecutionContext,
+	settings: ActionExecutionSettings<StudioConfig, ShowStyleConfig>,
+	actionId: string,
+	_userData: ActionPlayGraphics
```

Comment:
Underscore at the beginning usually means that you're not using that argument, but you are